assets/src/apps/authoring/BottomPanel.tsx

```diff
@@ -64,7 +64,8 @@ export const BottomPanel: React.FC<BottomPanelProps> = (props: BottomPanelProps)
   const debounceSaveChanges = useCallback(
     debounce(
       (activity) => {
-        dispatch(saveActivity({ activity, undoable: true }));
```

Comment:
By default, saveActivity now debounces the torus save.
I added an immediate mode for all areas of the app using saveActivity that I wasn't changing in this PR.